core/utils.py

```diff
@@ -232,6 +232,22 @@ def dict_from_yaml(yaml_str: str) -> Dict[str, Any]:
         raise InvalidInputException(e)
 
 
+def yaml_from_dict(dictionary: Dict[str, Any], width: int = 80) -> Any:
+    """Gets the YAML representation of a dict.
+
+    Args:
+        dictionary: dict. Dictionary for conversion into yaml.
+        width: int. Width for the yaml representation, default value
+            is set to be of 80.
+
+    Returns:
+        str. Converted yaml of the passed dictionary.
+    """
+    return yaml.dump(
+           dictionary, allow_unicode=True, width=width
+    )
```

Comment:
```suggestion
    return yaml.safe_dump(
       dictionary, allow_unicode=True, width=width
    )
```

Reply:
Also can you try to do `reveal_type(yaml.safe_dump(dictionary, allow_unicode=True, width=width))` and then run the mypy checks and send the output of it?

Reply:
Installed Mypy and stubs for third party libraries.
Starting Mypy type checks.
core/utils.py:246: note: Revealed type is 'Any'


Mypy type checks unsuccessful. Please fix the errors. For more information, visit: https://github.com/oppia/oppia/wiki/Backend-Type-Annotations

Reply:
this is the output

Reply:
Hmm, okay, you will need to add the ` # type: ignore[no-any-return]` after the `return yaml.safe_dump(`. And also add this comment above that line:

```python
    # The type ignore is needed, because typestubs define the return type
    # of 'safe_dump' as 'Any' which is wrong.
```

Reply:
added it